docs/pages/clients/development-workflows.md

```diff
@@ -0,0 +1,50 @@
+---
+
+
+## Iterating on your product with custom clients
+
+With a custom client, you can provide a stable environment to load any changes to your application that can be defined in JavaScript or other asset files that must only be built and installed once. Other updates to your application, whether defined directly in your ios/ and android/ directories or by modules or SDKs you choose to install, will require a new version of your client to take advantage of your changes.
+
+To enforce an API contract between the JavaScript and native portion of your application, you should set the [`runtimeVersion`](../distribution/runtime-versions.md) value in your app.json or app.config.js. This runtime version will be used to ensure your clients only load compatible version of your application in both development and production.
+
+## Tools
+
+### Tunnel URLs
+
+`expo start` can expose your development server on a publicly available URL that can be accessed through firewalls from around the globe.
+
+To get a tunneled URL, pass the `--tunnel` flag to `expo start` from the command line, or select the "tunnel" option for "CONNECTION" if you are using the developer tools.
+
+### Published Updates
+
+[`expo publish`](../workflow/publishing.md) packages the current state of your JavaScript and asset files into an optimized "update" stored on a free hosting service provided by Expo.  Published updates can be loaded in Expo clients without needing to check out a particular commit or leave a development machine running.
+
+### QR Codes
+
+You can use our endpoint to generate a QR code that can be easily loaded by a build of your custom development client.
+
+Requests to `https://qr.expo.dev/development-client`, when supplied the query parameters
+
+| parameter | value |
+| --------------- | ----------------------- |
+| `appScheme`         | URL-encoded deeplinking scheme of your client (defaults to `exp+{slug}` where slug is the value set in your app.json)       |
+| `url`         | URL of a bundle or manifest to load  (e.g. as provided by `expo publish`)     |
+
+receive a response with an SVG image containing a QR code that can be easily scanned to load a version of your project in your client.
+
+
+## Example Workflows
+
+These are a few examples of workflows to help your team get the most out of your custom clients.  If you come up with others that would be useful for other teams, please [submit a PR](https://github.com/expo/expo/blob/master/CONTRIBUTING.md#-updating-documentation) to share the knowledge!
+
+### Development Builds
+
+Developers on your team with expertise working with Xcode and Android Studio can update, review, and test changes to the native portion of your application and release them to your team periodically. The rest of your team can install these builds on their devices and simulators and quickly iterate on the JavaScript portion of your application without needing to understand and maintain the tooling required to create a new build.
```

Comment:
This feels like it's missing a little bit of "how" (especially in contrast with the PR Previews section, which fills in the "how" nicely). Is the intent to steer people towards EAS Build internal distribution?